src/calculateRentalCost.js

```diff
@@ -4,7 +4,23 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const SHORT_TERM = 3;
+  const SHORT_TERM_DISCOUNT = 20;
+  const LONG_TERM = 7;
+  const LONG_TERM_DISCOUNT = 50;
+  let discount = 0;
```

Comment:
Prefer 'const' over 'let' where possible. Since the 'discount' variable is being reassigned, consider restructuring the code to use 'const' for each conditional case.